src/NexyCrypt.php

```diff
@@ -299,7 +320,7 @@ private function signedPostRequest($uri, array $payload)
      *
      * @return ResponseInterface
      */
-    private function request($method, $uri, array $options = [])
+    private function request($method, $uri, array $options = ['verify' => false])
```

Comment:
I'm not a big fan of this.

If we pass an array without the `verify` key, this will simply not exist.

What is the goal of this option?